lib/tasks/configTenantAdmin.js

```diff
@@ -47,8 +47,9 @@ Command.run = function (options) {
                done();
             },
             questions,
+            writeEnvironment,
             updateSQLInsert,
-            updateTestConfig,
+            // updateTestConfig,
```

Comment:
so, do we not need this fn() any more?  See CodeQL warning below.

Reply:
no we don't, it patches files that no longer exist. removed now